src/main/java/org/umssdiplo/automationv01/core/managepage/employeedetail/EmployeeDetail.java

```diff
@@ -6,13 +6,21 @@
 import org.umssdiplo.automationv01.core.managepage.assignemployeeitem.AssignEmployeeItemModal;
 import org.umssdiplo.automationv01.core.utils.CommonEvents;
 
+import java.util.List;
+
 public class EmployeeDetail extends BasePage {
     @FindBy(id = "assigned-items-title")
     private WebElement asssignedActiveTitle;
 
     @FindBy(id = "assign-active")
     private WebElement assignItemButton;
 
+    @FindBy(id = "employee-detail-title")
+    private WebElement pageTitle;
+
+    @FindBy(css = ".mat-list-item.mat-3-line.ng-star-inserted")
+    private List<WebElement> assignedItems;
+
     public EmployeeDetail() {
         CommonEvents.isVisible(asssignedActiveTitle);
```

Comment:
assignedActiveTitle instead of asssignedActiveTitle

Reply:
Done